src/v0/destinations/google_adwords_offline_conversions/utils.test.js

```diff
@@ -273,3 +274,97 @@ describe('getClickConversionPayloadAndEndpoint util tests', () => {
     );
   });
 });
+
+describe('populateConsentForGoogleDestinations', () => {
+  // Returns an object with adUserData and adPersonalization properties set to UNSPECIFIED when no consents are provided
+  it('GOOGLE_ADWORDS_OFFLINE_CONVERSIONS : store sales conversion without any mention in integrations object', () => {
```

Comment:
Instead of repeating the destination name in each test case description, you can wrap all the test case groups under a `describe` that contains the destination name.